src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static String getRandomColor() {
```

Comment:
The `getRandomColor()` method is currently static. According to the checklist, static methods are generally considered bad practice. It's better to create an instance of the `ColorSupplier` class and call the method on that instance.